vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/ApplicationSteps.java

```diff
@@ -156,4 +158,15 @@ public void changeAppiumSettings(List<NamedEntry> settings)
 
         execute(executesMethod, entry("setSettings", prepareArguments("settings", prepareArguments(params, values))));
     }
+
+    /**
+     * Run application in background for given period of time.
+     * @param period  Total background run time according to
+     *                <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601</a> standard
+     */
+    @When("I run app in background for `$period` period")
```

Comment:
In all steps that do something with the app we don't use shortcuts in their wording, but I also noticed that there are place with not just application, but mobile application, so I think we should standardise wordings for mobile app steps

Reply:
`mobile application` -> 
```
When I send mobile application to background for `$period` period
```

Reply:
@uarlouski  please create an issue to refactor existing steps